### PR TITLE
buff to hos, rebalancing taser revolver and buffing of departmental head

### PR DESCRIPTION
**Pull request by projectkepler-ru**
## About The Pull Request
This PR massively redo everything I ever worked toward and balances everything around skyrat value while also having me modularise everything, files, codes has been moved to bubber modular folder in order to account for rebase or something and anything that I can't move or don't want to move remain in their respective folder

While the PR it self has by this point been tested by the players and myself with help of other players, it will impact balancing.

## Changelog

:cl:
server: modularised everything I added.. I think
add: a new m1911 pistol for the hop
add: the devil may cry  kit for the hos
add: paper notice for hos to know what is in each of their kits
add: new tech tree for the .460 ammo research
imageadd: m1911 pistol captain and devil maycry pistol sprites sprite
imageadd: new blueshield revolver sprite 
balance: significantly increased the cost of firing taser bolt from the blueshield energy revolver
balance: hos glock damage multiplier to 0.7 instead of 0.5
balance: TG taser bolt instead of Skyrat taser bolt, this instantly stun and down someone but deal no stamina damage.
balance: significantly reduced bleeding on NT rep pistol and PDH
balance: PDH now get rubber ammo to replace 2 of their lethal magazine, the stamina damage also got buffed to compensate while reducing down wounds even more
balance: .460 now doesnt cause as much bleeding but have increased projectiles speed and armor penetration
balance: buffed up the sabre to 21 damage since 15 was fucking horrendous with you needing to hit someone 10 time before they get downed
/:cl:

### CODE REVIEW

File: code/game/objects/items/melee/misc.dm [hunk 1]
```diff
@@ -67,7 +67,7 @@
 	righthand_file = 'icons/mob/inhands/weapons/swords_righthand.dmi'
 	flags_1 = CONDUCT_1
 	obj_flags = UNIQUE_RENAME
-	force = 15
+	force = 21 //Was told to buff this thing so let's go
```
Comment (KathrinBailey) [inline code/game/objects/items/melee/misc.dm:70]: no, modularise it according to guidelines.
```suggestion
//	force = 15	//Bubber Edit: Original
	force = 21
```

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 2]
```diff
@@ -17,7 +17,7 @@
 	new /obj/item/radio/headset/heads/captain/alt(src)
 	new /obj/item/radio/headset/heads/captain(src)
 	new /obj/item/storage/belt/sabre(src)
-	new /obj/item/storage/box/gunset/pdh(src) // SKYRAT EDIT CHANGE - ORIGINAL: new /obj/item/gun/energy/e_gun(src)
+	new /obj/item/storage/box/gunset/m1911_captains(src)//KEPLER EDIT
```
Comment (KathrinBailey) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm:20]: non-negotiable, no silly comments, it's bubber edit.
don't make it confusing like this

File: modular_zubbers/icons/obj/reshicaptainpistol.dmi
Comment (KathrinBailey) [inline modular_zubbers/icons/obj/reshicaptainpistol.dmi:1]: modularisation!
file name needs to be the same as what you'd find on TG. It's gotta be the name for the ballistic guns.

**Review by KathrinBailey — CHANGES_REQUESTED** [review]
follow modularisation guidelines

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm (hunk 2 above)
Comment (projectkepler-ru) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm:20]:
```suggestion
	new /obj/item/storage/box/gunset/m1911_captains(src)//BUBBERSTATION
```

File: code/game/objects/items/melee/baton.dm [hunk 3]
```diff
@@ -26,7 +26,7 @@
 	/// The length of the knockdown applied to the user on clumsy_check()
 	var/clumsy_knockdown_time = 18 SECONDS
 	/// How much stamina damage we deal on a successful hit against a living, non-cyborg mob.
-	var/stamina_damage = 35 // SKYRAT EDIT - Less Stamina Damage (Original: 55)
+	var/stamina_damage = 55 //KEPLER EDIT BACK TO 55, SKYRAT HAD IT 35 FOR SOME ATROCIOUS REASON
```
Comment (Cyprex) [inline code/game/objects/items/melee/baton.dm:29]: bubber edit

Comment (Cyprex) [inline code/game/objects/items/melee/baton.dm:29]:
```suggestion
	var/stamina_damage = 55 //BUBBERSTATION EDIT BACK TO 55, SKYRAT HAD IT 35 FOR SOME ATROCIOUS REASON
```

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 4]
```diff
@@ -17,7 +17,7 @@
 	new /obj/item/radio/headset/heads/captain/alt(src)
 	new /obj/item/radio/headset/heads/captain(src)
 	new /obj/item/storage/belt/sabre(src)
-	new /obj/item/storage/box/gunset/pdh(src) // SKYRAT EDIT CHANGE - ORIGINAL: new /obj/item/gun/energy/e_gun(src)
+	new /obj/item/storage/box/gunset/pdh(src)//BUBBERSTATION EDIT TAKE 2
```
Comment (Cyprex) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm:20]: I'm guessing you changed this in a previous PR & are only now updating the comment?

Comment (projectkepler-ru) [reply code/game/objects/structures/crates_lockers/closets/secure/security.dm:20]: Yeah take 2, they had PDH, then they didn't, then they did again.

**Comment by nevimer** [comment]
This is obsolete, I am closing it to clear up the PR open list. You can reopen it if you want :3 - too many merge conficts to do anything but close with for now.

**Pull request closed without merging**